dolphinscheduler-ui/src/service/modules/projects/index.ts

```diff
@@ -26,6 +26,14 @@ export function queryProjectListPaging(params: ListReq): any {
   })
 }
 
+export function queryProjectWithAuthorizedLevelListPaging(params: any): any {
```

Comment:
It's better to define the type for the params.

Reply:
Suggestions received and have been revised as suggested.